### PR TITLE
Fix cf_command argument for deploy tool

### PR DESCRIPTION
**Pull request by A-Shumway42**
*A note to PR reviewers: it may be helpful to review our [code review documentation](https://github.com/GSA/notifications-api/blob/main/docs/all.md#code-reviews) to know what to keep in mind while reviewing pull requests.*

## Description

Fix cg deploy tool so the proper argument is supplied with `cf_command`.

## Security Considerations
Make sure secrets are secured in push command.

### CODE REVIEW

**Review by ccostino — CHANGES_REQUESTED** [review]
Ugh, sadly the API version failed again, and at this point for lack of any other better option or easy way to test this, I say we refer to the GitHub values directly as was originally proposed (e.g., `${{ secrets.ENV_VAR_NAME }}`) directly in the new `push` command being defined.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -74,23 +75,7 @@ jobs:
...
-          --var LOGIN_DOT_GOV_LOGOUT_URL="$LOGIN_DOT_GOV_LOGOUT_URL"
-          --var LOGIN_DOT_GOV_BASE_LOGOUT_URL="$LOGIN_DOT_GOV_BASE_LOGOUT_URL"
-          --var LOGIN_DOT_GOV_SIGNOUT_REDIRECT="$LOGIN_DOT_GOV_SIGNOUT_REDIRECT"
-          --var LOGIN_DOT_GOV_INITIAL_SIGNIN_URL="$LOGIN_DOT_GOV_INITIAL_SIGNIN_URL"
+        cf_command: "push -f manifest.yml --vars-file deploy-config/staging.yml --var var-name=${{ secrets.DANGEROUS_SALT }} --var var-name=${{ secrets.SECRET_KEY }} --var var-name=${{ ADMIN_CLIENT_USERNAME=admin-notify }} --var var-name=${{ secrets.ADMIN_CLIENT_SECRET }} --var var-name=${{ secrets.NEW_RELIC_LICENSE_KEY }} --var var-name=${{ secrets.NR_BROWSER_KEY }} --var var-name=${{ COMMIT_HASH }} --var var-name=${{ secrets.LOGIN_PEM }} --var var-name=${{ LOGIN_DOT_GOV_CLIENT_ID }} --var var-name=${{ LOGIN_DOT_GOV_USER_INFO_URL }} --var var-name=${{ LOGIN_DOT_GOV_ACCESS_TOKEN_URL }} --var var-name=${{ LOGIN_DOT_GOV_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_BASE_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_SIGNOUT_REDIRECT }} --var var-name=${{ LOGIN_DOT_GOV_INITIAL_SIGNIN_URL }} --strategy rolling"
```
Comment (ccostino) [inline .github/workflows/deploy.yml:78]: D'oh!  We do need to do something different for the `ADMIN_CLIENT_USERNAME` here...

I think `--var ADMIN_CLIENT_USERNAME=admin-notify` will work here.  The rest should be okay!

```suggestion
        cf_command: "push -f manifest.yml --vars-file deploy-config/staging.yml --var var-name=${{ secrets.DANGEROUS_SALT }} --var var-name=${{ secrets.SECRET_KEY }} --var ADMIN_CLIENT_USERNAME=admin-notify --var var-name=${{ secrets.ADMIN_CLIENT_SECRET }} --var var-name=${{ secrets.NEW_RELIC_LICENSE_KEY }} --var var-name=${{ secrets.NR_BROWSER_KEY }} --var var-name=${{ COMMIT_HASH }} --var var-name=${{ secrets.LOGIN_PEM }} --var var-name=${{ LOGIN_DOT_GOV_CLIENT_ID }} --var var-name=${{ LOGIN_DOT_GOV_USER_INFO_URL }} --var var-name=${{ LOGIN_DOT_GOV_ACCESS_TOKEN_URL }} --var var-name=${{ LOGIN_DOT_GOV_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_BASE_LOGOUT_URL }} --var var-name=${{ LOGIN_DOT_GOV_SIGNOUT_REDIRECT }} --var var-name=${{ LOGIN_DOT_GOV_INITIAL_SIGNIN_URL }} --strategy rolling"
```

**Review by ccostino — CHANGES_REQUESTED** [review]
One more quick change, @A-Shumway42!

**Review by ccostino — APPROVED** [review]
Thanks, @A-Shumway42

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -75,24 +76,32 @@ jobs:
         cf_password: ${{ secrets.CLOUDGOV_PASSWORD }}
         cf_org: gsa-tts-benefits-studio
         cf_space: notify-staging
-        push_arguments: >-
+        cf_command: >-
+
+
```
Comment (ccostino) [inline .github/workflows/deploy.yml:81]: We can get rid of these two lines. :-)

```suggestion
```

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -75,24 +76,32 @@ jobs:
...
+          --var NOTIFY_E2E_TEST_EMAIL="$NOTIFY_E2E_TEST_EMAIL"
+          --var NOTIFY_E2E_TEST_PASSWORD="$NOTIFY_E2E_TEST_PASSWORD"
+          --var LOGIN_DOT_GOV_REGISTRATION_URL="$LOGIN_DOT_GOV_REGISTRATION_URL"
           --var LOGIN_DOT_GOV_CLIENT_ID="$LOGIN_DOT_GOV_CLIENT_ID"
           --var LOGIN_DOT_GOV_USER_INFO_URL="$LOGIN_DOT_GOV_USER_INFO_URL"
           --var LOGIN_DOT_GOV_ACCESS_TOKEN_URL="$LOGIN_DOT_GOV_ACCESS_TOKEN_URL"
           --var LOGIN_DOT_GOV_LOGOUT_URL="$LOGIN_DOT_GOV_LOGOUT_URL"
           --var LOGIN_DOT_GOV_BASE_LOGOUT_URL="$LOGIN_DOT_GOV_BASE_LOGOUT_URL"
-          --var LOGIN_DOT_GOV_SIGNOUT_REDIRECT="$LOGIN_DOT_GOV_SIGNOUT_REDIRECT"
+          --VAR LOGIN_DOT_GOV_SIGNOUT_REDIRECT="$LOGIN_DOT_GOV_SIGNOUT_REDIRECT"
```
Comment (ccostino) [inline .github/workflows/deploy.yml:99]: Let's keep it lower case for consistency - there's nothing special about this one! :-)

```suggestion
          --var LOGIN_DOT_GOV_SIGNOUT_REDIRECT="$LOGIN_DOT_GOV_SIGNOUT_REDIRECT"
```

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -75,24 +76,32 @@ jobs:
...
           --var LOGIN_DOT_GOV_INITIAL_SIGNIN_URL="$LOGIN_DOT_GOV_INITIAL_SIGNIN_URL"
           --var LOGIN_DOT_GOV_CERTS_URL="$LOGIN_DOT_GOV_CERTS_URL"
+          --var LOGIN_PEM="$LOGIN_PEM"
+          --strategy rolling
+
```
Comment (ccostino) [inline .github/workflows/deploy.yml:104]: One more extra newline that's not needed:

```suggestion
```

**Review by ccostino — COMMENTED** [review]
Thank you, @A-Shumway42!  Just a few minor edit suggestions and this part is good!

We still need to figure out the egress proxy part, so let's continue to hold for a moment.

Also, thank you for alphabetizing everything! 🙂

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (A-Shumway42) [reply .github/workflows/deploy.yml:99]: Oops! I looked over it and I suppose the caps blended in 😅

**Review by ccostino — APPROVED** [review]
I think we're good to give this a go on the admin side now that the API side has worked!

**Pull request merged**